test/input_files/v4.3/passed/passed_symbolic_duplicates.vcf

```diff
@@ -0,0 +1,4 @@
+##fileformat=VCFv4.3
```

Comment:
Do we need 4.1 and 4.2 test files?

Reply:
I don't think they are needed because there would be no change in the input files except for the version number as the specs don't diverge for this particular example. However, I noticed that there are other inputs where this would be true as well but they were added. So for the sake of completeness I am adding test files for 4.1 and 4.2.